### ISSUE TITLE
Option to disable movable top panel on main page

### ISSUE THREAD

**Issue by Bugaddr** [Iamlooker/Droid-ify#32]
Sorry i don't know what the part / element should be named. Please check the images below, i request a toggle to disable the movement of the topmost part (one with droid-ify written) & the one on the single app description page. The topmost part movement makes navigation unnecessarily time-taking on small phones

1:
![IMG_20220710_161720.jpg](https://user-images.githubusercontent.com/99622785/178141910-685ee5c7-7e45-43e3-a507-99279cfb30fe.jpg)

2:
![IMG_20220710_162459.jpg](https://user-images.githubusercontent.com/99622785/178141972-a68ec8b9-8e90-42a3-b579-23ff2aa2b3c1.jpg)

**Comment by Iamlooker** [comment]
I think you have created the same issues previously, it was closed right?

**Comment by Bugaddr** [comment]
> I think you have created the same issues previously, it was closed right?

No, this is my first issue regarding this. Maybe you got confused with #24

**Comment by Iamlooker** [comment]
No I mean the same issue about TopAppBar

**Comment by Bugaddr** [comment]
> No I mean the same issue about TopAppBar

I think had opened 3 issues afaik, please point out if there are any.

Thanks

![IMG_20220712_020125.jpg](https://user-images.githubusercontent.com/99622785/178353041-0bcbc460-c9b3-4ac9-b8af-2b55ec14ecb5.jpg)

**Comment by Bugaddr** [comment]
And yeah, sorry i forgot to add this one also, the small space below search bar appears every time when we search.

![IMG_20220712_042043.jpg](https://user-images.githubusercontent.com/99622785/178372652-b4edad4e-4f86-4c92-aa95-1320a982c042.jpg)

I think this is something that got unnoticed at the early implementation.

Thanks

**Comment by Iamlooker** [comment]
> I think i had opened 3 issues afaik, please point out if there are any.

Ooh my bad

**Comment by Bugaddr** [comment]
Alternatively, we can combine this (disabling top header movement) with already present "List animations" toggle & renaming it to "disable extra motion/movement", or something else which suits the case.

**Comment by comradekingu** [comment]
Does that animation or use of space do anything for anyone?
Default off seems like the sane option if it is to be included at all (?)
